test/lottery.test.js

```diff
@@ -131,4 +134,28 @@ contract('Lottery', (accounts) => {
 
     });
 
+    it("batch mint multiple tickes", async () => {
+        const LotteryContract = this.lotteryProxy.methods;
+        const async = [
+            await LotteryContract.newBatchLotteryItem([alice,bob,carol],[1, 2, 3],[10 ** 5, 10 ** 4, 10 ** 3]),
+            await LotteryContract.balanceOfBatch(
```

Comment:
there was no balanceOfBatch and newBatchLotteryItem in your upgrade lottery contract